example/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Quicktube</title>
```

Comment:
Once this is merged, would be cool to have this on the `gh-pages` branch so it's available online as a demo!